charts/ziti-edge-tunnel/Chart.yaml

```diff
@@ -1,10 +1,10 @@
 apiVersion: v2
-appVersion: 0.22.20
+appVersion: 0.22.22
```

Comment:
We can go ahead and pull in 0.22.23 if you're ready for it.

Reply:
done